pubsub/README.rst

```diff
@@ -139,6 +139,23 @@ any metrics agent that implements the same interface as ``MetricsAgent``
   in a single request.
 
 
+If you prefer pull-based metrics like Prometheus you will be pleased to know that
+the subscriber records Prometheus metrics of the form ``gcloud_aio_<metric>``,
+which will be ignored if you don't use Prometheus and will be automatically
+included if you do:
+
+- ``pubsub_consume`` (labels: ``outcome = {'succeeded', 'cancelled', 'failed',
+  'failfast'}``) - [counter] a consume operation has completed with a given
+  outcome
+- ``pubsub_consume_latency`` (labels: ``aspect = {'receive', 'runtime'}``) -
+  [histogram] how many seconds taken to receive a message, or callback runtime
+- ``pubsub_batch_failed`` (labels: ``component = {'acker', 'nacker'}``) -
```

Comment:
Thoughts on making this a generic `pubsub_batch` metric with `outcome = {'succeeded', 'failed'}`?

Reply:
I'm good with that!